sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -594,8 +589,9 @@ def get_user_client(self, user):
 
         :param user: The ID (name), dict representing the properties or :class:`UserProxy`
             instance of the user to be retrieved.
+        :type user: Union[str, UserProxy, Dict[str, Any]]
         :returns: A `UserProxy` instance representing the retrieved user.
-        :rtype: ~azure.cosmos.UserProxy
+        :rtype: ~azure.cosmos.user.UserProxy
```

Comment:
Let's just use the base namespace - we want to avoid sending people to the subnamespaces....
```suggestion
        :rtype: ~azure.cosmos.UserProxy
```